front/src/common/Map/Search/useSearchOperationalPoint.tsx

```diff
@@ -122,7 +119,7 @@ export default function useSearchOperationalPoint({
 
   useEffect(() => {
     if (debouncedSearchTerm) {
-      searchOperationalPoints();
+      searchOperationalPoints(debouncedSearchTerm).then((results) => setSearchResults(results));
```

Comment:
using the promise syntax seems a good alternative to introducing an async function inside a useEffect